src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_AMOUNT = 5;
+
+    public Figure getRandomFigure() {
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();
+        Circle circle = new Circle();
+        circle.setName("Circle");
+        circle.setRadius(random.nextInt(100));
+        circle.setColor(supplier.getRandomColor());
+
+        IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
+        isoscelesTrapezoid.setName("IsoscelesTrapezoid");
+        isoscelesTrapezoid.setColor(supplier.getRandomColor());
+        isoscelesTrapezoid.setHeight(random.nextInt(100));
+        isoscelesTrapezoid.setSideOne(random.nextInt(100));
+        isoscelesTrapezoid.setSideTwo(random.nextInt(100));
+        isoscelesTrapezoid.setColor(supplier.getRandomColor());
+
+        Rectangle rectangle = new Rectangle();
+        rectangle.setName("Rectangle");
+        rectangle.setSideOne(random.nextInt(100));
+        rectangle.setSideTwo(random.nextInt(100));
+        rectangle.setColor(supplier.getRandomColor());
+
+        RightTriangle rightTriangle = new RightTriangle();
+        rightTriangle.setName("RightTriangle");
+        rightTriangle.setFirstLeg(random.nextInt(100));
+        rightTriangle.setSecondLeg(random.nextInt(100));
+        rightTriangle.setColor(supplier.getRandomColor());
+
+        Square square = new Square();
+        square.setName("Square");
+        square.setSide(random.nextInt(100));
```

Comment:
it's not the good practice to do like that, consider using switch case to generate random figure using constructor, not setters